.vscode/settings.json

```diff
@@ -22,5 +22,14 @@
     ],
     "mocha.env": {
         "TS_NODE_TYPE_CHECK": false
+    },
```

Comment:
Make sure to exclude these once you are finished working on this.

Reply:
crap. sorry these keep getting in.